tests/data/constVelocityLineScan.json

```diff
@@ -1,111 +1,87 @@
-     1000.0, 0.0, 8.0,
-     1000.0, 0.0, 4.0,
-     1000.0, 0.0, 0.0,
-     1000.0, 0.0, -4.0,
-     1000.0, 0.0, -8.0,
-     1000.0, 0.0, -12.0,
-     1000.0, 0.0, -16.0
-   ],
-   "CENTER_EPHEMERIS_TIME": 1000.0,
-   "INT_TIME": 0.1,
-   "NUMBER_OF_QUATERNIONS": 9,
-   "ITRANSS": [
-     0,
-     0,
-     10.0
-   ],
-   "IKCODE": -99999,
-   "PLATFORM_ID": "UNKNOWN",
-   "ISIS_Z_DIRECTION": 1,
-   "MAX_VALID_HT": 20,
-   "TRI_PARAMETERS": [
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     0,
-     50.0,
-     0,
-     0
-   ],
-   "TRAJ_ID": "UNKNOWN",
-   "ITRANSL": [
-     0,
-     10.0,
-     0
-   ],
-   "REF_DATE_TIME": "UNKNOWN"
+  "model_name" : "USGS_ASTRO_LINE_SCANNER_SENSOR_MODEL",
+  "center_ephemeris_time": 1000.0,
+  "starting_ephemeris_time": 992.0,
+  "line_scan_rate": [
+    [0.5, 992.0, 0.1]
+  ],
+  "detector_sample_summing": 1,
+  "detector_line_summing": 1,
+  "t0_ephemeris": -8.0,
+  "dt_ephemeris": 0.2,
+  "t0_quaternion": -8.0,
+  "dt_quaternion": 0.2,
+  "focal2pixel_lines": [0.0, 10.0, 0.0],
+  "focal2pixel_samples": [0.0, 0.0, 10.0],
+  "focal_length_model": {
+    "focal_length": 50,
+    "epsilon": 1.0
+  },
+  "image_lines": 16,
+  "image_samples": 16,
+  "detector_center" : {
+    "line" : 0.5,
+    "sample" : 7.5
+  },
+  "interpolation_method": "lagrange",
+  "optical_distortion": {
+    "radial": {
+      "coefficients": [0.0, 0.0, 0.0]
+    }
+  },
+  "radii": {
+    "semimajor": 10,
+    "semiminor": 10,
+    "unit":"m"
+  },
+  "reference_height": {
+    "maxheight": 1,
+    "minheight": -1,
+    "unit": "m"
+  },
+  "sensor_location": {
+    "unit": "m",
+    "locations": [
+      [1000.0, 0.0, 16.0],
+      [1000.0, 0.0, 12.0],
+      [1000.0, 0.0, 8.0],
+      [1000.0, 0.0, 4.0],
+      [1000.0, 0.0, 0.0],
+      [1000.0, 0.0, -4.0],
+      [1000.0, 0.0, -8.0],
+      [1000.0, 0.0, -12.0],
+      [1000.0, 0.0, -16.0]
+    ]
+  },
+  "sensor_orientation": {
+    "quaternions": [
+      [0.0, -0.707106781186547, 0.0, 0.707106781186547],
+      [0.0, -0.707106781186547, 0.0, 0.707106781186547],
+      [0.0, -0.707106781186547, 0.0, 0.707106781186547],
+      [0.0, -0.707106781186547, 0.0, 0.707106781186547],
+      [0.0, -0.707106781186547, 0.0, 0.707106781186547],
+      [0.0, -0.707106781186547, 0.0, 0.707106781186547],
+      [0.0, -0.707106781186547, 0.0, 0.707106781186547],
+      [0.0, -0.707106781186547, 0.0, 0.707106781186547],
+      [0.0, -0.707106781186547, 0.0, 0.707106781186547]
+    ]
+  },
+  "sensor_velocity": {
+    "unit": "m",
+    "velocities": [
+      [0.0, 0.0, -20.0],
+      [0.0, 0.0, -20.0],
+      [0.0, 0.0, -20.0],
+      [0.0, 0.0, -20.0],
+      [0.0, 0.0, -20.0],
+      [0.0, 0.0, -20.0],
+      [0.0, 0.0, -20.0],
+      [0.0, 0.0, -20.0],
+      [0.0, 0.0, -20.0]
+    ]
+  },
+  "starting_detector_line": 0,
+  "starting_detector_sample": 0,
+  "sun_position": [100.0, 100.0, 0.0],
```

Comment:
Should `sun_position` and `sun_velocity` use the same objects are `sensor_velocity`. That should help with unit management, etc.

Reply:
:+1: